src/backend/columnar/README.md

```diff
@@ -0,0 +1,324 @@
+# Introduction
+
+Citus Columnar offers a per-table option for columnar storage to
+reduce IO requirements though compression and projection pushdown.
+
+# Design Trade-Offs
+
+Existing PostgreSQL row tables work well for OLTP:
+
+* Support `UPDATE`/`DELETE` efficiently
+* Efficient single-tuple lookups
+
+The Citus Columnar tables work best for analytic or DW workloads:
+
+* Compression
+* Doesn't read unnecessary columns
```

Comment:
wonder if we should incorporate chunk/block filtering here as well.

Reply:
We can expand on it a bit; I just wanted a quick introduction for now.